package.json

```diff
@@ -5,8 +5,9 @@
   "main": "server.js",
   "scripts": {
     "test": "jest",
-    "start": "node server.js",
-    "db": "mysql -u root -p"
+    "start": "nodemon server.js",
+    "db": "mysql -u root -p",
+    "seeds": "node db/seeds.js"
```

Comment:
This file doesn't have to be run it auto runs when the db connection is made at the bottom of server.js